wagtailnhsukfrontend/templates/wagtailnhsukfrontend/skip_link.html

```diff
@@ -0,0 +1 @@
+<a class="nhsuk-skip-link" href={{ href|default:"#maincontent" }}> {{ text|default:"Skip to main content" }} </a>
```

Comment:
You've got no quotes around the href attribute! (`href=#maincontent` should be `href="#maincontent"`)